sections/advanced/server-side-rendering.js

```diff
@@ -56,34 +56,35 @@ const ServerSideRendering = () => md`
   import { ServerStyleSheet } from 'styled-components'
 
   export default class MyDocument extends Document {
-    render() {
+    static getInitialProps ({ renderPage }) {
       const sheet = new ServerStyleSheet()
-      const main = sheet.collectStyles(<Main />)
+      const page = renderPage(App => props => sheet.collectStyles(<App {...props} />))
       const styleTags = sheet.getStyleElement()
+      return {
+        ...page, 
+        styleTags
+      }
+    }
 
+    render() {
       return (
         <html>
           <Head>
             {/* ... */}
 
-            {styleTags}
+            {this.props.styleTags}
           </Head>
-
           <body>
-            <div className="root">
-              {main}
-            </div>
-
+            <Main />
             <NextScript />
           </body>
         </html>
       )
     }
   }
   \`\`\`
-
-  Here we're wrapping the \`Main\` component, which contains the Next.js app, and are using this
-  to extract the styles on the server side.
+ 
+  > Refer to [Next.js example `with-styled-components`](https://github.com/zeit/next.js/tree/master/examples/with-styled-components)
```

Comment:
* the

And this won't need to be a note (Remove `> `)

Also, maybe we can make this a little nicer? "Refer to [our example] in the next.js repo for an up-to-date usage example."

Reply:
Thanks, @philpl. I've updated the language.

Reply:
I've removed the leading `>`. Should be good now.